### PR TITLE
Added new projects

### PR DESCRIPTION
**Pull request by alirizaadiyahsi**
Added a new web framework, 2 new libraries and a project template

### CODE REVIEW

File: dotnet-developer-projects.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
   * [BeYourMarket](https://github.com/beyourmarket/beyourmarket) - BeYourMarket is a peer-to-peer marketplace framework http://beyourmarket.com 
 
 ## Web Frameworks
+  * [ASP.NET Boilerplate](https://github.com/aspnetboilerplate/aspnetboilerplate) - ASP.NET Boilerplate is a general purpose application framework specially designed for new modern web applications. It uses already familiar tools and implements best practices around them to provide you a SOLID development experience.
```
Comment (rpetrusha) [inline dotnet-developer-projects.md]: general purpose --> general-purpose
SOLID --> solid

File: dotnet-developer-projects.md [hunk 2]
```diff
@@ -107,6 +108,8 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
   * [Automatonymous](https://github.com/MassTransit/Automatonymous) - An asynchronous state machine engine with messaging support. 
   * [Bing.RestClient](https://github.com/AdvancedREI/Bing.RestClient) - Access the Bing REST APIs from a unified client.
   * [BoxKite.Twitter](https://github.com/NickHodge/BoxKite.Twitter/) - Twitter .NET Client Library for the 1.1 Twitter API, incorporating REST API, User streaming and Search Streaming. Uses Reactive Extensions (Rx).
+  * [Castle.LoggingFacility.MsLogging](https://github.com/volosoft/castle-logging-ms-adapter) - Castle Logging Adapter for Microsoft.Extensions.Logging Package
```
Comment (rpetrusha) [inline dotnet-developer-projects.md]: Package --> package. (not that this should end with a period)]
Can you provide any additional detail?

File: dotnet-developer-projects.md [hunk 3]
```diff
@@ -107,6 +108,8 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
...
+  * [Castle Windsor ASP.NET Core / Microsoft.Extensions.DependencyInjection Adapter](https://github.com/volosoft/castle-windsor-ms-adapter) - This library is a Castle Windsor adapter for Microsoft.Extensions.DependencyInjection nuget package. It works for ASP.NET Core and other type of applications.
```
Comment (rpetrusha) [inline dotnet-developer-projects.md]: adapter for --> adapter for the
nuget --> NuGet 
other type --> other types 
What other types of applications does it work for?

File: dotnet-developer-projects.md [hunk 4]
```diff
@@ -380,6 +383,7 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
   * [Myrtille](https://github.com/cedrozor/myrtille) - A native HTML4/5 Remote Desktop Protocol client, HTTP gateway written in C# for .NET
  
 ## Project Templates
+  * [ASP.NET Core & Mvc/Angular5 Startup Template](https://github.com/aspnetboilerplate/module-zero-core-template) - This is a template to create ASP.NET Core MVC / Angular based startup projects for [ASP.NET Boilerplate](https://github.com/aspnetboilerplate/aspnetboilerplate). This project also support Multi-Tenancy.
```
Comment (rpetrusha) [inline dotnet-developer-projects.md]: support --> supports
Multi-Tenancy --> multi-tenancy

**Comment by alirizaadiyahsi** [comment]
@rpetrusha, I made the changes that you suggest.

**Pull request merged**